patterns/templates/jolla-hw-adaptation-@DEVICE@.yaml

```diff
@@ -8,23 +8,34 @@ Requires:
 - droid-hal-@DEVICE@
 - droid-hal-@DEVICE@-img-boot
 - droid-hal-@DEVICE@-kernel-modules
-- droid-config-@DEVICE@-sailfish-config
+- droid-config-@DEVICE@-sailfish
 - droid-config-@DEVICE@-pulseaudio-settings
 - droid-config-@DEVICE@-policy-settings
 - droid-config-@DEVICE@-preinit-plugin
+- droid-hal-version-@DEVICE@
+
+- pulseaudio-modules-droid
 
 # These need to be per-device due to differing backends (fbdev, eglfs, hwc, ..?)
 - qt5-qtwayland-wayland_egl
 - qt5-qpa-hwcomposer-plugin
 - qtscenegraph-adaptation
 
+# GStreamer v0.10 multimedia support is being deprecated
 #- gstreamer0.10-droidcamsrc
 #- gstreamer0.10-omx
 #- gstreamer0.10-droideglsink
 #- nemo-qtmultimedia-plugins-gstvideotexturebackend
```

Comment:
No, because current Sailfish OS releases still run gst0.10. If a porter doesn't add an extra OBS repository with gst-droid, at least he can fallback to 0.10 (like mako, or any other 10.1 port)

Once 0.10 is deprecated in a public sfos release, will clean up these